README.md

```diff
@@ -18,27 +18,21 @@ All of this results in a more iterative process for developing manifests, one th
 
 ## Install
 
-> You should have Go installed *(minimum version 1.8.1)*. If not, follow the instructions in the [official installation guide](https://golang.org/doc/install#install).
```

Comment:
It might still be nice to have an overview statement, for example:

> The ksonnet CLI, `ks`, can be installed in three different ways. Choose the method that best matches your setup: